src/DataWrangling/metadata.jl

```diff
@@ -26,15 +26,28 @@ Keyword Arguments
 =================
 - `dataset`: Supported datasets are `ECCO2Monthly()`, `ECCO2Daily()`, `ECCO4Monthly()`, `EN4Monthly(),
              `RepeatYearJRA55()`, or `MultiYearJRA55()`.
-- `dates`: The dates of the dataset, in a `AbstractCFDateTime` of `Dates.AbstractDateTime` format. Note
-           this can either be a range or a vector of dates, representing a time-series. For a single date,
-           use [`Metadatum`](@ref).
+- `dates`: The dates of the dataset (`Dates.AbstractDateTime` or `CFTime.AbstractCFDateTime`).
+           Note this can either be a range or a vector of dates, representing a time-series.
+           For a single date, use [`Metadatum`](@ref).
+- `start_date`: If `dates = nothing`, we can prescribe the first date of metadata as a date
+                (`Dates.AbstractDateTime` or `CFTime.AbstractCFDateTime`). `start_date` should lie
+                within the date range of the dataset. Default: nothing.
+- `end_date`: If `dates = nothing`, we can prescribe the last date of metadata as a date
+              (`Dates.AbstractDateTime` or `CFTime.AbstractCFDateTime`). `end_date` should lie
+                within the date range of the dataset. Default: nothing.
 - `dir`: The directory where the dataset is stored.
 """
 function Metadata(variable_name;
                   dataset,
                   dates = all_dates(dataset, variable_name),
-                  dir = default_download_directory(dataset))
+                  dir = default_download_directory(dataset),
+                  start_date = nothing,
+                  end_date = nothing)
+
+    if !isnothing(start_date) && !isnothing(end_date)
+        @info "Slicing date range within $start_date and $end_date"
```

Comment:
I'm worried we are getting carried away with info statements, ClimaOcean is really chatty right now

Reply:
fair, deleting it